### PR TITLE
[Object] Dalaran Fountain

### PR DESCRIPTION
**Pull request by Knindzagxg**
Transfering from SC by @FrancescoBorzi request.

Update for Dalaran Fountain.

### CODE REVIEW

**Comment by Helias** [comment]
what this PR does? move the fountain of its position?

**Comment by Knindzagxg** [comment]
For some reason its missing from DB and it corrects how it should stand.
WoWScrnShot_083120_162743

File: data/sql/updates/pending_db_world/rev_1598881542523907700.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1598881542523907700');
+/*
+ * General: Dalaran Fountain
+ * Update by Knindza | <www.azerothcore.org>
+ * Copyright (C) <www.shadowburn.net> & <www.lichbane.com>
+*/
+
+/* Content 3.3.5 */
+
+REPLACE INTO `gameobject` (`guid`, `id`, `map`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`, `VerifiedBuild`) VALUES (268968, 191446, 571, 1, 1, 5804.72, 640.837, 647.627, -0.680677, 0, 0, -0.333807, 0.942641, 180, 255, 1, 0);
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1598881542523907700.sql:10]: better to use DELETE/INSERT instead of REPLACE

Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1598881542523907700.sql]: Sorted.

**Comment by Geekena** [comment]
tested sql
no problems found fountain is in center

**Comment by Krath** [comment]
After applying, I see two fountains. 
Is this correct?

**Comment by Knindzagxg** [comment]
Make sure to have GM off, since second one is for Arthas death event.

File: data/sql/updates/pending_db_world/rev_1598881542523907700.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+/*
+ * General: Dalaran Fountain
+ * Update by Knindza | <www.azerothcore.org>
+ * Copyright (C) <www.shadowburn.net> & <www.lichbane.com>
+*/
+
+/* Content 3.3.5 */
+
+DELETE FROM `gameobject` WHERE `guid`=268968;
+INSERT INTO `gameobject` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`, `ScriptName`, `VerifiedBuild`) VALUES (268968, 191446, 571, 0, 0, 1, 1, 5804.72, 640.837, 647.627, -0.680677, 0, 0, -0.333807, 0.942641, 180, 255, 1, '', 0);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1598881542523907700.sql:11]:
```suggestion
INSERT INTO `gameobject` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`, `ScriptName`, `VerifiedBuild`) VALUES 
(268968, 191446, 571, 0, 0, 1, 1, 5804.72, 640.837, 647.627, -0.680677, 0, 0, -0.333807, 0.942641, 180, 255, 1, '', 0);
```

**Review by Kitzunu — CHANGES_REQUESTED** [review]
This GUID is already IN USE by another object!

Please make sure to always use a free GUID when you add new Gameobjects!

gameobject
---
```sql
SELECT * FROM `gameobject` WHERE `guid`=268968
```
| guid | id | map | zoneId | areaId | spawnMask | phaseMask | position_x | position_y | position_z | orientation | rotation0 | rotation1 | rotation2 | rotation3 | spawntimesecs | animprogress | state | ScriptName | VerifiedBuild | 
| ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | --- | ---: | 
| 268968 | 181371 | 0 | 0 | 0 | 1 | 1 | -4699,79 | -1222,77 | 501,659 | -1,37881 | 0 | 0 | 0 | 0 | 300 | 0 | 1 |  | 0 |

File: data/sql/updates/pending_db_world/rev_1598881542523907700.sql (hunk 2 above)
Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1598881542523907700.sql]: Sorted xP

**Comment by Knindzagxg** [comment]
Nice find Kit, i totally forgot to check GUID.
Sorted.

**Review by Kitzunu — CHANGES_REQUESTED** [review]
image

Also there are two fountains, you need to add some kind of condition to swap them before and after LK kill :)

**Comment by Knindzagxg** [comment]
Yeah, will check more about this one xP

**Comment by FrancescoBorzi** [comment]
closing as discussed on Discord

**Pull request closed without merging**